tlsr9/drivers/B95/usbhw.h

```diff
@@ -337,7 +337,7 @@ static inline void usbhw_set_eps_en(usb_ep_en_e ep)
     }
     else
     {
-        reg_usb_edp_en |= ep;
+        reg_usb_edp_en = ep;

```

Comment:
This will disable the enabled endpoints.

Reply:
follow the driver in new ble sdk